### DISCUSSION TITLE
Total Size Plots

### DISCUSSION THREAD

**Discussion by RRCHA000** [Chia-Network/chia-blockchain#6268] [Support]
I have 82 K-32 plots and the total size is 8.117 TiB. There is something wrong. How can I find invalid plots?
![plots](https://user-images.githubusercontent.com/84117578/120201871-46e0f780-c1fc-11eb-8bbd-f8b08c2f9a1c.png)

**Comment by XIII39** [comment]
The plot size is correct.
Each k=32 plot is 101.4 GiB <-- _this is not GB, it is GiB_
82 plots = 8,314.8 GiB

1 TiB = 1024 GiB
Therefore 8,314.8 GiB / 1,024 = 8.119921875 TiB

**Reply by room101-dev** [reply]
FUD designed system

A plot is 108gb on your HDD, at best a formatted drive yields 90% utilization

A 6tb drive allows me to store 51 plots. Done that's it. Extrapolate from there.

K=32, is just a debug switch to that DEV could set K=1 and make small plots in minutes to test the system, now the default is k=32, nothing else matters, and on the disk it occupy's 108gb, 108.7 & 108.6 are common.

**Comment by BrandtH22** [comment]
@RRCHA000 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
